tests/test_loadflow.py

```diff
@@ -131,3 +132,16 @@ def test_provider_parameters():
     assert LoadFlowComponentStatus.MAX_ITERATION_REACHED == result[0].status
     assert 6 == result[0].iteration_count
 
+def test_run_lf_with_report():
+    n = pp.network.create_ieee14()
+    reporter = rp.Reporter()
+    report1 = str(reporter)
```

Comment:
maybe we can have an assertion on the string representation of a just initialized reporter

Reply:
done